test-app/tsconfig.json

```diff
@@ -21,36 +21,20 @@
     "module": "es6",
     "experimentalDecorators": true,
     "paths": {
-      "dummy/tests/*": [
+      "test-app/tests/*": [
         "tests/*"
       ],
-      "dummy/*": [
-        "tests/dummy/app/*",
+      "test-app/*": [
         "app/*"
       ],
-      "tracked-built-ins": [
```

Comment:
@linainu should we make this

```json
"tracked-built-ins": [
  "../addon/addon"
]
```

instead of deletion?

Reply:
@SergeAstapov I wasn't sure if we still needed this because we have tracked-built-ins as a dependency